spec/defines/puppet_authorization_spec.rb

```diff
@@ -1,109 +1,132 @@
+# frozen_string_literal: true
+
 require 'spec_helper'
 describe 'puppet_authorization' do
   let(:facts) do
-    { :concat_basedir => '/dne' }
+    { concat_basedir: '/dne' }
   end
   let(:title) do
     '/tmp/foo'
   end
 
   context 'defaults' do
-    it {is_expected.to contain_concat(title).with({
-      :path    => title,
-      :replace => false,
-    })}
-
-    it { is_expected.to contain_concat__fragment("00_header_#{title}").with({
-      :target => title,
-    }).with_content(/authorization: \{\n  rules: \[\]\n/)}
-
-    it { is_expected.to contain_concat__fragment("99_footer_#{title}").with({
-      :target => title,
-    }).with_content(/\}\n/)}
-
-    it { is_expected.to contain_hocon_setting("authorization.version.#{title}").
-          that_requires("Concat[#{title}]").with({
-      :path    => title,
-      :setting => 'authorization.version',
-      :value   => 1,
-    })}
-
-    it { is_expected.to contain_hocon_setting(
-                            "authorization.allow-header-cert-info.#{title}").
-                            that_requires("Concat[#{title}]").with({
-      :path    => title,
-      :setting => 'authorization.allow-header-cert-info',
-      :value   => false,
-    })}
+    it {
+      is_expected.to contain_concat(title).with({
+                                                  path: title,
+                                                  replace: false,
+                                                })
+    }
+
+    it {
+      is_expected.to contain_concat__fragment("00_header_#{title}").with({
+                                                                           target: title,
+                                                                         }).with_content(%r{authorization: \{\n  rules: \[\]\n})
```

Comment:
Similar here: you can use `.with_target(title)`. You can also spread it over multiple lines
```suggestion
      is_expected.to contain_concat__fragment("00_header_#{title}")
        .with_target(title)
        .with_content(%r{authorization: \{\n  rules: \[\]\n})
```

Reply:
The update Layout/FirstHashElementIndentation style handled this.  It's a different solution, but its legible.